server/controllers/wazuh-utils.ts

```diff
@@ -126,4 +127,47 @@ export class WazuhUtilsCtrl {
       return ErrorResponse(error.message || error, 3036, 500, response);
     }
   }
+
+
+    /**
+   * Returns Wazuh frontend logs
+   * @param {Object} context 
+   * @param {Object} request
+   * @param {Object} response
+   * @returns {Array<String>} app logs or ErrorResponse
+   */
+    async getFrontendLogs(context: RequestHandlerContext, request: KibanaRequest, response: KibanaResponseFactory) {
+      try {
+        const lastLogs = await read(
```

Comment:
to avoid confusion with the response attribute you could rename `lastLogs` with `rawLogs`